app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1>List of Posts</h1>
\ No newline at end of file
+<section class="container">
```

Comment:
- [ ] [OPTIONAL] Currently, the counters aren't updated in the database and thus appear wrong in the views as shown in this image:
![image](https://github.com/hmunish/ruby-rails-blog/assets/24703171/a676a7aa-e5fe-4610-b681-dc6ae9c533f7)

So kindly use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the update counter methods in the models so every time new data has been saved to the database the counter will be updated.